cartographer_ros/cartographer_ros/assets_writer.cc

```diff
@@ -1,105 +0,0 @@
-/*
- * Copyright 2016 The Cartographer Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#include "cartographer_ros/assets_writer.h"
-
-#include "cartographer/common/make_unique.h"
-#include "cartographer/common/time.h"
-#include "cartographer/io/file_writer.h"
-#include "cartographer/io/null_points_processor.h"
-#include "cartographer/io/ply_writing_points_processor.h"
-#include "cartographer/io/points_processor.h"
-#include "cartographer/io/xray_points_processor.h"
-#include "cartographer/mapping/proto/trajectory.pb.h"
-#include "cartographer/mapping_2d/proto/range_data_inserter_options.pb.h"
-#include "cartographer_ros/map_writer.h"
```

Comment:
1. Start Cartographer (offline or online)
2. Start the Occupancy Grid node.
3. use `map_saver` from http://wiki.ros.org/map_server